package.json

```diff
@@ -15,9 +12,16 @@
         "url": "https://github.com/algeriatech/jokes-cli/issues"
     },
     "keywords": [
-        "joke.dz",
-        "cli"
+            "joke.dz",
+            "cli"
     ],
+    "bin": {
+        "jokes-cli": "./jokes-cli.js"
+    },
+    "scripts": {
+        "jokes-cli": "./jokes-cli.js",
+        "start": "node app.js"
+    },
     "dependencies": {
         "commander": "^2.9.0",
         "figlet": "^1.2.0",
```

Comment:
I know, you did not add those dependencies, but I think they are not used, are they?